### PR TITLE
docs(hero-banner): update usage page - FRONT-1175

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

Update usage page of hero banner (only the anatomy table).

Reference: https://citnet.tech.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Hero+banner

### CODE REVIEW

File: src/website/src/pages/ec/components/banners/hero-banner/docs/usage.mdx [hunk 1]
```diff
@@ -24,11 +24,12 @@ Hero banner images have an aspect ratio of 3:1, meaning the width will be always
...
 ![hero banner 2](https://inno-ecl.s3.amazonaws.com/media/images/EC/Hero%20Banner/Hero%20Banner02.png)
 
-| Components     | Constraint | Restrictions                                               |
-| -------------- | ---------- | ---------------------------------------------------------- |
-| hero image     | optional   | choice between 2 background colours when there is no image |
-| description    | mandatory  | 100-150 characters maximum, including punctuation          |
-| call to action | mandatory  | 3 words – 25 characters maximum, including punctuation     |
+| Components            | Use      | Requirement                                                | Link |
+| --------------------- | -------- | ---------------------------------------------------------- | ---- |
+| title                 | optional | single sentence, between 40-50 characters including space  | no   |
```
Comment (planctus) [inline src/website/src/pages/ec/components/banners/hero-banner/docs/usage.mdx]: not sure it really makes any difference, @emeryro , but the order is different in https://citnet.tech.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Hero+banner,  hero image is the first... :)

File: src/website/src/pages/ec/components/banners/hero-banner/docs/usage.mdx [hunk 2]
```diff
@@ -24,11 +24,12 @@ Hero banner images have an aspect ratio of 3:1, meaning the width will be always
...
+| hero image            | optional | choice between 2 background colours when there is no image | no   |
+| description           | optional | 100-150 characters maximum, including space                | no   |
+| call to action button | optional | actionable, verbs, 25 characters maximum, including space  | yes  |
 
 ## When to use
```
Comment (planctus) [inline src/website/src/pages/ec/components/banners/hero-banner/docs/usage.mdx:34]: There are differences in the "sections" below, not sure whether this is supposed to be updated as here:
image

**Comment by emeryro** [comment]
Line order updated.
The description of the ticket clearly states that only the anatomy should be modified, so I assume other sections are out of scope

**Review by planctus — APPROVED** [review]
I approve the changes, we are just waiting for confirmation from the designers that the anatomy table was the only element in the page to be modified.

**Comment by emeryro** [comment]
doc approved by designers, merging

**Pull request merged**